src/how-to-start-contributing.md

```diff
@@ -0,0 +1,134 @@
+[crates-io-repo]: https://github.com/rust-lang/crates.io
+[crates-io-guide]: https://github.com/rust-lang/crates.io/blob/main/docs/CONTRIBUTING.md
+[crates-io-issues]: https://github.com/rust-lang/crates.io/issues
+[bors-repo]: https://github.com/rust-lang/bors
+[bors-guide]: https://github.com/rust-lang/bors/blob/main/docs/development.md
+[bors-issues]: https://github.com/rust-lang/bors/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22
+[rustc-perf-repo]: https://github.com/rust-lang/rustc-perf
+[rustc-perf-issues]: https://github.com/rust-lang/rustc-perf/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22help%20wanted%22
+[triagebot-repo]: https://github.com/rust-lang/triagebot
+[triagebot-issues]: https://github.com/rust-lang/triagebot/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22
+[playground-repo]: https://github.com/rust-lang/rust-playground
+[playground-issues]: https://github.com/rust-lang/rust-playground/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22help%20wanted%22
+[rustlings-repo]: https://github.com/rust-lang/rustlings
+[rustlings-guide]: https://github.com/rust-lang/rustlings/blob/main/CONTRIBUTING.md
+[rustlings-issues]: https://github.com/rust-lang/rustlings/issues
+[mdbook-repo]: https://github.com/rust-lang/mdBook
+[mdbook-guide]: https://github.com/rust-lang/mdBook/blob/master/CONTRIBUTING.md
+[mdbook-issues]: https://github.com/rust-lang/mdBook/labels/E-Help-wanted
+
+If you want to get inspired, check all the [rust-lang][rust-lang-repos] repositories!
+
+[rust-lang-repos]: https://github.com/orgs/rust-lang/repositories?type=all&q=sort%3Astars
+
+### Different kinds of contributions
+
+There are various ways in which you can contribute to Rust projects:
+
+- Writing code is the most obvious one. However, it does not have to be only in Rust! Even though
+  most of our projects are of course written in Rust, we also use other technologies. For example,
+  you can help improve our GitHub CI workflows, automation Python scripts or contribute to web
+  frontends with HTML/CSS/JS (e.g. [Rustdoc][rustdoc-ui] or [Benchmark suite website][rustc-perf-ui]).
+  Play to your strengths!
+- Improving documentation is possibly one of the easiest starting points for contributions.
+  Did you find a typo, something that was unclear, or do you miss some useful information on this
+  page, elsewhere in the Forge or in some other Rust documentation? Great, then send a pull request
+  and you'll become a contributor! :)
+- Improving tests is also very valuable, as there is never enough of them. This can include also
+  documenting existing tests or writing tests for already fixed issues that lack a proper test.
+- You can also help with improving issue trackers of our repositories, by helping with triaging issues
+  or by reproducing old issues to find out whether they are still valid or not.
```

Comment:
Suggestion: backlink to the Issue Triage page?

Reply:
I wanted to avoid repo-specific things on this page. But I guess that issue triaging on r-l/r is similar to other repos, so let's link it.